velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -324,6 +324,25 @@ std::unique_ptr<ParquetTypeWithId> ReaderBase::getParquetColumnInfo(
           const auto& child = children[0];
           auto type = child->type();
           isRepeated = true;
+          if (schemaElement.converted_type == thrift::ConvertedType::LIST &&
```

Comment:
I think it would be easier to read if we move this section to just the `case thrift::ConvertedType::LIST:` section and remove the `schemaElement.converted_type == thrift::ConvertedType::LIST` check there, since it's only for lists. Add a `[[fallthrough]];` at the end in that section so that the more generic scenario will still be handled with existing code

Reply:
I have rearranged the code as suggested please take a look.